README.md

```diff
@@ -28,10 +28,28 @@ After you upgrade, you'll need to merge back any customizations you made to `App
 
 *Otherwise*
 
-* Download the latest version of [Application Insights Java SDK](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started), [scroll down to section](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started)and follow the instructions to manually download the SDK and replace the old Jars.
+* Download the latest version of [Application Insights Java SDK](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started), [scroll down to the getting started section](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started) and follow the instructions to manually download the SDK and replace the old `.jar` files.
  
 Compare the old and new `ApplicationInsights.xml`. Many of the changes you see are because we added and removed modules. Reinstate any customizations that you made.
 
+## Application Insights for Java Roadmap
+The Application Insights team have been hard at work to deliver the next wave of features for our Java support and experience. Below is an outline of the features and improvements that are planned for our next release, which is targeted to be completed for mid-year 2018.
```

Comment:
This needs to be revised. We will possibly have multiple releases until mid-year 2018. Stable release is days away. After that, the goal is to release whenever there are substantial bug fixes or new features.

Reply:
Yes, we plan to have a frequent releasing cadence. Infact we would like to get fixes out as soon as we accumulate enough that it quantifies for a minor release.

Reply:
I'll clarify that this isn't going to all happen in one big-bang 💥 release.